osm2streets/src/lanes/mod.rs

```diff
@@ -87,6 +88,47 @@ impl LaneType {
         }
     }
 
+    /// Determines if the lane is a travel lane that is represented in OSM `*:lanes` tags.
+    /// Note that the `lanes` tag counts car driving lanes, excluding bike lanes, whereas the
+    /// `:lanes` suffix specifies that each lane, including bike lanes, should have a value between
+    /// `|`s. This function identifies the latter kind.
```

Comment:
I don't know if you were aware of this, it's not very well presented on the wiki and has big implications in `osm2lanes` parsing.